ee/recommendation/ml_service/api.py

```diff
@@ -0,0 +1,63 @@
+from fastapi import FastAPI, Depends
+from utils import pg_client
+from core.model_handler import recommendation_model
+from utils.declarations import FeedbackRecommendation
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from crons.base_crons import cron_jobs
+from auth.auth_key import api_key_auth
+from core import feedback
+from fastapi.middleware.cors import CORSMiddleware
+
+
+app = FastAPI()
+app.schedule = AsyncIOScheduler()
+
+origins = [
+    "*"
+]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+@app.on_event('startup')
```

Comment:
please use Lifespan instead of setup/shutdown https://fastapi.tiangolo.com/advanced/events/